### ISSUE TITLE
Server Error

### ISSUE THREAD

**Issue by decadance-dance** [voxel51/fiftyone#1405]
### Describe the problem
When I use the zoo model to predict the new bbox, I get the error
![image](https://user-images.githubusercontent.com/86170544/140934696-47340abb-997e-4924-ab2a-69a5660416d4.png)
After the prediction, I can see new bboxes and interact with images (everything is the same as usual), but I cannot access filtering by class either from the application or from the code. 
And when I click on "filter by label" i get this error.
### Code to reproduce issue

```
import fiftyone as fo
import fiftyone.zoo as foz
from fiftyone import ViewField as F

dataset = foz.load_zoo_dataset(
        "open-images-v6",
        split="train",
        label_types=['detections'],
        classes = ["Motorcycle", "Bicycle"],
        max_samples=20,
        seed=2,
        shuffle=True,
        dataset_name="cycles"
    )
stage = fo.FilterLabels("detections", F("label").is_in(["Motorcycle", "Bicycle", "Car", "Bus", "Track", "Person"]))
view = dataset.add_stage(stage)

model = foz.load_zoo_model("faster-rcnn-resnet50-fpn-coco-torch")
view.apply_model(model, label_field="predictions")

session.view = view
```

### What areas of FiftyOne does this bug affect?

-   [+] `App`: FiftyOne application issue
-   [+] `Core`: Core `fiftyone` Python library issue
-   [+] `Server`: Fiftyone server issue

**Comment by benjaminpkane** [comment]
Are you using colab?

**Comment by decadance-dance** [comment]
> Are you using colab?
yes

**Comment by decadance-dance** [comment]
> Ok. Resolved by #1392. We should have a release this week or next. Installing from source in colab can also be done. See [here](https://github.com/voxel51/fiftyone#source-installs-in-google-colab)

ty
